addOns/fuzz/src/main/resources/org/zaproxy/zap/extension/fuzz/resources/Messages.properties

```diff
@@ -387,8 +387,6 @@ fuzz.httpfuzzer.description = Allows to fuzz HTTP messages.
 fuzz.httpfuzzer.messagetype = HTTP
 fuzz.httpfuzzer.fuzzerNamePrefix = HTTP - {0}
 
-fuzz.httpfuzzer.searcher.name = HTTP Fuzzer Results
```

Comment:
Why was this removed? It's used in ExtesionHttpFuzzer.java

Reply:
Yes, there were 2 copies of the same key/value pair.

Reply:
Confirmed, thanks!
![image](https://user-images.githubusercontent.com/7570458/69349716-3edaeb80-0c46-11ea-9dd8-1603bed8b3a7.png)